definitions/procedures/backup/offline/candlepin_db.rb

```diff
@@ -1,53 +0,0 @@
-module Procedures::Backup
-  module Offline
-    class CandlepinDB < ForemanMaintain::Procedure
-      metadata do
-        description 'Backup Candlepin DB offline'
-        tags :backup
-        label :backup_offline_candlepin_db
-        for_feature :candlepin_database
-        preparation_steps { Checks::Candlepin::DBUp.new unless feature(:candlepin_database).local? }
-        param :backup_dir, 'Directory where to backup to', :required => true
-        param :tar_volume_size, 'Size of tar volume (indicates splitting)'
```

Comment:
It's funny this was defined, but never passed to the procedure (heck, even the CLI param is called `--split-pulp-tar`), so yay less unused complexity?